image-viewer.py

```diff
@@ -1,37 +1,40 @@
-from tkinter import Tk, Label, Button
+from tkinter import Tk, Label, Button, Menu, filedialog
 from PIL import Image, ImageTk
 from os import listdir
 from os.path import isfile
 
 class ImageViewer():
     def __init__(self):
+                     
         self.current_image_index = 0
         self.image_files = []
+        self.blank_img = Image.new('RGBA',(500,300),(0,0,0,0)) # Blank Image to display if no other images are found   
+        self.img_dir = "./images" # Directory to search images in -- by default is the ./images folder
```

Comment:
To relate to my previous comment, I guess this is one of those where you wouldn't necessarily know that "./images" is the default directory, so the comment helps, but there is an alternative:
```suggestion
        defaultDirectory = "./images"
        self.img_dir = "./images"
```

This makes it clear what "./images" is without needing a comment!